### PR TITLE
fix(case overview) Handle cases started by system

### PR DESCRIPTION
**Pull request by DumitruCorini**
covers [BPO-236](https://bonitasoft.atlassian.net/browse/BPO-236)

### CODE REVIEW

File: uid-pages/form-case-overview/src/caseoverview.json [hunk 1]
```diff
@@ -1406,11 +1406,24 @@
         "return  (!$data.openCase && $data.archivedCase.length > 0) ? $data.archivedCase[0] : $data.openCase;"
       ],
       "exposed" : false
+    },
+    "initiator" : {
```
Comment (julienmege) [inline uid-pages/form-case-overview/src/caseoverview.json]: This Code should be Factorized with the code of "Task executed by System"

File: uid-pages/form-case-overview/src/assets/json/localization.json [hunk 2]
```diff
@@ -150,8 +153,10 @@
     "Process": "Proceso",
     "No Events yet.": "Ningúna actividad.",
     "Nothing noticeable happened yet... But it surely will come!": "Ninguna tarea ejecutada o disponible de momento.",
-    "Case started": "Caso iniciado",
```
Comment (julienmege) [inline uid-pages/form-case-overview/src/assets/json/localization.json]: Shouldn't be deleted

File: uid-pages/form-case-overview/test/specs/formCaseOverview.feature [hunk 3]
```diff
@@ -22,4 +22,86 @@ Feature: Form case overview
     And The open case business data is defined
     When I visit the open case index page
     Then The correct BDM headers are visible
-    And The incorrect BDM headers don't exist
\ No newline at end of file
+    And The incorrect BDM headers don't exist
+
```
Comment (julienmege) [inline uid-pages/form-case-overview/test/specs/formCaseOverview.feature]: Maybe should remove the "i don't see" assertions

File: uid-pages/form-case-overview/src/caseoverview.json (hunk 1 above)
Comment (DumitruCorini) [reply uid-pages/form-case-overview/src/caseoverview.json]: :heavy_check_mark:

File: uid-pages/form-case-overview/test/specs/formCaseOverview.feature (hunk 3 above)
Comment (DumitruCorini) [reply uid-pages/form-case-overview/test/specs/formCaseOverview.feature]: :heavy_check_mark:

File: uid-pages/form-case-overview/src/assets/json/localization.json (hunk 2 above)
Comment (DumitruCorini) [reply uid-pages/form-case-overview/src/assets/json/localization.json]: :heavy_check_mark:

**Pull request merged**